### ISSUE TITLE
Would be great to make some tutorial with every step explained including configuring AWS

### ISSUE THREAD

**Issue by progerzua** [williamparry/cloudRIG#70]

**Comment by williamparry** [comment]
Agree. Ideally you shouldn't need a tutorial because it takes care of it, but the initial grab of the AWS creds, and Parsec server id etc can be confusing and there's little that can be done around it.

I'll put it on the website (when I get around to making it)

**Comment by progerzua** [comment]
Thank you, man)

By the way, where you`ve got: 

> Stream games and applications for approx. $0.13/hr

Parsec tutorial talks about g2.2xLarge and g.3.4xLarge instances, that starting from 0.77$/hr.
I`m not into AWS instances, just opened it and confused by long list of different types).

By the way, don`t you need any help with project, I`m into Python and a liitle bit into Javascript?

**Comment by williamparry** [comment]
For sure. The way cloudRIG works is that it uses Spot instances, which are cheaper, but you can be booted off at any time since it uses the excess capacity of AWS. If you set it up on Parsec then they use the on-demand version that you don't have the risk of being booted off from.

Where cloudRIG is useful is that it will auto save your machine in the case of you being booted off (or if you stop it), so you can change zone and boot it up again or just wait till the price drops and try again.

Help on the project would be awesome! Some areas would be testing the binaries before they are released (I only have Linux atm and there's a big update I want to push out), general bug fixing, documentation, etc, so take your pick :)

**Comment by progerzua** [comment]
Sounds interesting. I`m testing Snoost service right now, its pretty cheap but sooo buggy(actually its small startup maintained by 3 guys, so...) =(

Can`t promise anything, but if i`ll have some free time, it would be interesting to explore repo and do something useful\write some wiki.

**Comment by williamparry** [comment]
Parsec is great. They have a really active Discord and their team is friendly.

If you're keen on helping out, I can give you a test account for you to use AWS/cloudRIG for free.

**Comment by progerzua** [comment]
> Parsec is great. They have a really active Discord and their team is friendly.

Parsec is totally great and mind-blowing(still surprised that its completely free), but Snoost service is not so good and still in Alpha.

> If you're keen on helping out, I can give you a test account for you to use AWS/cloudRIG for free.

Sounds great. I want to test Paperspace first, to compare results then) Looks like it have [better upload speed](https://www.reddit.com/r/cloudygamer/comments/6sefg1/parsecpaperspace_joy/) (I think so).
